test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/RequestSizeLimitResourceFilterTest.cs

```diff
@@ -0,0 +1,162 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Features;
+using Microsoft.AspNetCore.Mvc.Abstractions;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.Logging.Testing;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Mvc.Internal
+{
+    public class RequestSizeLimitResourceFilterTest
+    {
+        [Fact]
+        public void SetsMaxRequestBodySize()
+        {
+            // Arrange
+            var requestSizeLimitResoureFilter = new RequestSizeLimitResourceFilter(NullLoggerFactory.Instance);
+            requestSizeLimitResoureFilter.Bytes = 12345;
+            var resourceExecutingContext = CreateResourceExecutingContext(new IFilterMetadata[] { requestSizeLimitResoureFilter });
+
+            var httpMaxRequestBodySize = new TestHttpMaxRequestBodySizeFeature();
+            resourceExecutingContext.HttpContext.Features.Set<IHttpMaxRequestBodySizeFeature>(httpMaxRequestBodySize);
+
+            // Act
+            requestSizeLimitResoureFilter.OnResourceExecuting(resourceExecutingContext);
+
+            // Assert
+            Assert.Equal(12345, httpMaxRequestBodySize.MaxRequestBodySize);
+        }
+
+        [Fact]
+        public void SkipsWhenOverridden()
+        {
+            // Arrange
+            var requestSizeLimitResoureFilter = new RequestSizeLimitResourceFilter(NullLoggerFactory.Instance);
+            requestSizeLimitResoureFilter.Bytes = 12345;
+            var disableRequestSizeLimitResourceFilter = new DisableRequestSizeLimitResourceFilter(NullLoggerFactory.Instance);
+            var resourceExecutingContext = CreateResourceExecutingContext(new IFilterMetadata[] { requestSizeLimitResoureFilter, disableRequestSizeLimitResourceFilter});
+
+            var httpMaxRequestBodySize = new TestHttpMaxRequestBodySizeFeature();
+            resourceExecutingContext.HttpContext.Features.Set<IHttpMaxRequestBodySizeFeature>(httpMaxRequestBodySize);
+
+            // Act
+            requestSizeLimitResoureFilter.OnResourceExecuting(resourceExecutingContext);
+
+            // Assert
+            Assert.Null(httpMaxRequestBodySize.MaxRequestBodySize);
+        }
+
+        [Fact]
+        public void LogsFeatureNotFound()
```

Comment:
We haven't normally written unit tests for the logging when it's simple like this